src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        Random random = new Random();
+        int randomIndex = random.nextInt(colors.length);
+        return colors[randomIndex].toString();
```

Comment:
use `name()` instead of `toString()` for enum 
check [common mistakes file](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md) for this task